alcs-frontend/src/environments/environment.prod.ts

```diff
@@ -10,6 +10,7 @@ export const environment = {
   apiUrl: (window.env?.['apiUrl'] || 'http://localhost:8080') + '/alcs',
   authUrl: window.env?.['apiUrl'] || 'http://localhost:8080',
   homeUrl: window.env?.['homeUrl'] || '/home',
+  portalUrl: window.env?.['portalUrl'] || 'http://localhost:4201',
```

Comment:
is it mapped on actual dev env?

Reply:
I have updated the template js files with the value and will make an MR on devops repo with the values